lib/Progress/VariantSummaryFormatter.php

```diff
@@ -80,19 +80,11 @@ public function formatVariant(Variant $variant): string
     {
         $data = $this->paramProvider->provideFor($variant);
 
-        if (!$this->initialized) {
-            $this->initialize();
-        }
+        $this->normalNode = $this->parser->parse($variant->getSubject()->getFormat() ?? $this->format);
+        $this->baselineNode = $this->parser->parse($variant->getSubject()->getFormat() ?? $this->baselineFormat);
```

Comment:
this is a big performance regression (aside from the pointless assign). The parsing should be cached to avoid needing to initialize things here..